### ISSUE TITLE
Geany debugger can't open source code from folders with non-ascii names

### ISSUE THREAD

**Issue by Yanpas** [geany/geany-plugins#259]
Mine code is located at /home/user/Загрузки but when I run geany debugger it show error "Can't find source code file /home/user/\320\227\320\260\320\263\321\200\321\203\320\267\320\272\320\270/natural.cxx"

**Comment by b4n** [comment]
Maybe you could try #202, which should most probably fix this -- and if it doesn't I'd like to know :)

**Comment by Yanpas** [comment]
Could you build amd64 package please? (guess geany-debugger-plugin will e enough) I can't build due to bunch of dependencies and old versions of packages in my Trusty Ubuntu :(

**Comment by Yanpas** [comment]
Are your commits from #202 pulled to main geany-plugins repo?

**Comment by Yanpas** [comment]
Built on virtual machine. Bug still exists
![- 29 06 2015 - 00 03 53](https://cloud.githubusercontent.com/assets/7345761/8398325/63a6e30c-1df2-11e5-8bb5-b5930fb04982.png)

**Comment by Yanpas** [comment]
Screenshot showing version. Downloaded from your @b4n  repo, not from this
![- 29 06 2015 - 00 06 00](https://cloud.githubusercontent.com/assets/7345761/8398335/b9a3a2fe-1df2-11e5-89e5-c0093651bb5c.png)

**Comment by b4n** [comment]
> Downloaded from your @b4n repo, not from this

You used the appropriate branch?  'cause while I didn't test Cyrillic, I tired various special UTF8 characters and it worked OK…  but indeed I might have missed something

**Comment by Yanpas** [comment]
As far as I remember I used https://github.com/geany/geany-plugins
Right down the correct link and I will test again tommorow :)

**Comment by Yanpas** [comment]
Or this :/ 
https://github.com/b4n/geany-plugins

**Comment by b4n** [comment]
You need to try the `debugger/safer-gdb-mi-parsing` from https://github.com/b4n/geany-plugins/:
```console
$ git remote add b4n https://github.com/b4n/geany-plugins/
$ git fetch b4n debugger/safer-gdb-mi-parsing
$ git checkout -b b4n-debugger/safer-gdb-mi-parsing b4n/debugger/safer-gdb-mi-parsing
```

**Comment by Yanpas** [comment]
White I've made
`git clone https://github.com/b4n/geany-plugins.git`
`cd geany-plugins/`
`git remote add b4n https://github.com/b4n/geany-plugins/`
`git fetch b4n debugger/safer-gdb-mi-parsing`
`git checkout -b b4n-debugger/safer-gdb-mi-parsing b4n/debugger/safer-gdb-mi-parsing`
`./autogen.sh `
`cd debugger/`
`make`
`sudo checkinstall -D``
Well, bug isn't reproducable, seems to be fixed. Thanks!

**Comment by Yanpas** [comment]
But the debugger is untranslated and fully in English. IS it Ok?

**Comment by b4n** [comment]
> Well, bug isn't reproducable, seems to be fixed. Thanks!

Yay, great :)

> But the debugger is untranslated and fully in English. IS it Ok?

Well… the version of the translations in this branch aren't very recent for sure.  You can probably improve this by merging `master` into `b4n-debugger/safer-gdb-mi-parsing`:
```console
$ git checkout b4n-debugger/safer-gdb-mi-parsing
$ git merge master
```

----
But ultimately I should merge this branch into the main repository, as the original author agreed as he doesn't care anymore about the plugin.  I'm just a bit uneasy as I don't really use the plugin myself so could have overlooked some things, but if you use it and tell me it works fine/report issues, I guess it'd be enough :)

**Comment by Yanpas** [comment]
Well, these commands and rebuilding didn't help, still English. Close this issue when you will merge your branch to https://github.com/geany/geany-plugins please. 
Maybe that's because I've compiled only debugger, I can't compile all geany plugins due to https://gist.github.com/Yanpas/990a2d8f98e13b448faa

**Comment by b4n** [comment]
> Maybe that's because I've compiled only debugger

Ah yeah; to get translations you also need to build and install in the *po/* directory, it holds the translations.

> I can't compile all geany plugins due to https://gist.github.com/Yanpas/990a2d8f98e13b448faa

Ah, this is because some plugins depend on the development version of Geany (but don't properly check for it yet).
You can disable those when configuring:
```console
$ ./configure --disable-geanygendoc --disable-projectorganizer --disable-geanyprj
```
and then all should build fine.

**Comment by Yanpas** [comment]
yes, now bug seems absolutely fixed :) Now I can continue learning programming, thanks a lot again. 
Do you have permission to merge your brunch to this repo?

**Comment by b4n** [comment]
Fixed by #202
